vars/fcosKola.groovy

```diff
@@ -52,26 +55,26 @@ def call(params = [:]) {
 
         try {
             if (!params['skipBasicScenarios']) {
-                shwrap("cd ${cosaDir} && cosa kola run ${rerun} --basic-qemu-scenarios")
+                shwrap("cd ${cosaDir} && cosa kola run ${rerun} --output-dir=${outputDir} --basic-qemu-scenarios")
             }
-            shwrap("cd ${cosaDir} && cosa kola run ${rerun} --build=${buildID} ${arch} ${platformArgs} --parallel ${parallel} ${args} ${extraArgs}")
+            shwrap("cd ${cosaDir} && cosa kola run ${rerun} --output-dir=${outputDir} --build=${buildID} ${arch} ${platformArgs} --parallel ${parallel} ${args} ${extraArgs}")
         } finally {
-            shwrap("tar -c -C ${cosaDir}/tmp kola | xz -c9 > ${env.WORKSPACE}/kola.tar.xz")
+            shwrap("tar -c -C ${outputDir} kola | xz -c9 > ${env.WORKSPACE}/kola.tar.xz")
             archiveArtifacts allowEmptyArchive: true, artifacts: 'kola.tar.xz'
         }
         // sanity check kola actually ran and dumped its output in tmp/
-        shwrap("test -d ${cosaDir}/tmp/kola")
+        shwrap("test -d ${outputDir}/kola")
```

Comment:
Also, shouldn't this be just

```suggestion
        shwrap("test -d ${outputDir}")
```

now?

And similarly below.

Reply:
not exactly what you suggested, but I reworked it to fix this issue.